drivers/sensor/tdk/icm42670/Invn/InvError.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @defgroup InvError Error code
+ *	@brief    Common error code
+ *
+ *	@ingroup EmbUtils
+ *	@{
+ */
```

Comment:
this file still doesn't follow Zephyr conventions (doxygen not used in internal headers, UpperCase naming, etc.)

Reply:
Here I removed the @ingroup comments not relevant in this integration. Are there other changes you requested?